File: submissions/0782-jewels-and-stones/solution.ts
function numJewelsInStones(jewels: string, stones: string): number { 
  let count:number = 0;
  const jewelsMap = new Map();
    
    for (const jewel of jewels){
        jewelsMap.set(jewel, (jewelsMap.get(jewel) || 0) + 1);
    }
    
    for(const stone of stones){
        if(jewelsMap.has(stone)){
            count += 1
        }
    }
    
    return count;
};

File: submissions/0782-jewels-and-stones/solution.py
from collections import Counter
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        countStones = Counter(stones)
        ans = 0
        
        for jewel in jewels:
            if jewel in countStones:
                ans += countStones[jewel]
                
        return ans        
